core/domain/prod_validation_jobs_one_off.py

```diff
@@ -1013,7 +1031,7 @@ def _get_external_id_relationships(cls, item):
         return {
             'collection_ids': (
                 collection_models.CollectionModel,
-                [item.id[:item.id.find('-')]]),
+                [item.id[:item.id.rfind(base_models.VERSION_DELIMETER)]]),
```

Comment:
DELIMITER, here and below. (From "limit")

Reply:
Done.